.travis.yml

```diff
@@ -0,0 +1,16 @@
+dist: xenial
+
+langauge: python
+
+python: "3.6"
+
+before_install:
+  - sudo apt-get update
+
+install:
+  - sudo apt install python3-pip 
+  - sudo pip3 install -r requirements.txt
```

Comment:
This can be removed, let's just install the dependencies needed to run the linters which are in test-requirements.txt.

Reply:
Okay